package.json

```diff
@@ -14,15 +14,15 @@
     "start": "npm run storybook",
     "storybook": "start-storybook -p 6006 -s src/static",
     "build-storybook": "build-storybook -s ./src/static",
-    "test": "npm run test-lint && prettier --check './src/**/*' && node ./scripts/build.test.js && npm run test-jest && npm run test-chrome",
+    "test": "npm run test-lint && prettier --check './src/**/*' && node ./scripts/build.test.js && npm run test-jest && npm run test-ui",
     "test-lint": "eslint src/",
-    "test-chrome": "sh scripts/testcafe-chrome.sh",
-    "test-ie": "sh scripts/testcafe-ie.sh",
     "test-jest": "jest --env=jsdom",
+    "test-playwright": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 jest --config=jest.ui.config.js",
     "test-jest-coverage": "jest --env=jsdom --coverage",
     "test-jest-watch": "jest --env=jsdom --watch -o",
-    "test-ui": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe chrome ./**/ui.test.js",
-    "test-ui-hl": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe 'chrome:headless' ./**/ui.test.js",
+    "test-ui": "sh scripts/test-ui-testrunner.sh",
+    "test-ui-isolated": "TEST_HOST_STORYBOOK_URL=http://localhost:6006 testcafe chrome ./**/ui.test.js",
```

Comment:
isolated? Would rename that to test-ui-testcafe

Reply:
Yeah good point. I didn't really know what the best name would be. Those tasks won't run as long as storybook is not started separately.

I would go with "test-testcafe", ok? Or `test-testcafe-serverless` (and rename the playwrtight one too)?

Reply:
"test-testcafe", ok